demo/ops/config/application.properties

```diff
@@ -16,11 +16,18 @@ logging.config=logback-spring.xml
 # The default value is 10.
 thread.pool.size=10
 
+# org.sdo.epid.epid-online-url (optional) : URL
+# The URL of the Epid Verification Service.
+# If this property is set, the online verification of EPID keys is performed
+# by making a request to this URL.
```

Comment:
Please add an example on how this property would be set.

Reply:
Done